### ISSUE TITLE
Maintaining

### ISSUE THREAD

**Issue by unicod3** [storborg/packagetrack#4]
Is there still a maintainer for this project?

**Comment by storborg** [comment]
No, not really. I'm not actively using it or adding any new features. I've seen various forks come and go, but none seem to have stuck.

**Comment by unicod3** [comment]
I forked the @JosephDev 's version and made python3 changes. Fedex plugin also has some changes. 
Anyway thank you for the repo then. It is a great repository I was able to port it in my code with a few changes.

**Comment by alertedsnake** [comment]
I also [forked this myself](https://github.com/alertedsnake/packagetracker) because of the disinterest around here.  I made the python3 changes and started doing a whole lot of reworking, since I wasn't happy with the idea of the Package class.  I also completed implementation of USPS and FedEx tracking.
